lib/config/rules/style.js

```diff
@@ -0,0 +1,58 @@
+// Style:
+// These rules enforce consistent style across your codebase
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces vertical alignment. <FEEDBACK>
+  align: false,
+  // Requires using either ‘T[]’ or ‘Array' for arrays. <FEEDBACK>
+  'array-type': false,
+  // Requires parentheses around the parameters of arrow function definitions.
+  'arrow-parens': true,
+  // An interface or literal type with just a call signature can be written as a function type.
+  'callable-types': false,
+  // Enforces PascalCased class and interface names.
+  'class-name': true,
+  // Enforces formatting rules for single-line comments.
+  'comment-format': [true, 'check-space'],
+  // Enforces documentation for important items be filled out.
+  'completed-docs': false,
+  // Enforces a certain header comment for all files, matched by a regular expression.
+  'file-header': false,
+  // Ensures proper spacing between import statement keywords
+  'import-spacing': true,
+  // Requires interface names to begin with a capital ‘I’ <FEEDBACK>
+  'interface-name': false,
+  // Prefer an interface declaration over a type literal (`type T = { ... }`)
+  'interface-over-type-literal': false,
+  // Enforces basic format rules for JSDoc comments. <FEEDBACK>
+  'jsdoc-format': false,
+  // Requires parentheses when invoking a constructor via the `new` keyword.
+  'new-parens': true,
+  // Requires the use of `as Type` for type assertions instead of `<Type>`. <FEEDBACK>
+  'no-angle-bracket-type-assertion': false,
+  // Disallows one or more blank lines in a row. <FEEDBACK>
+  'no-consecutive-blank-lines': [true, 2],
+  // Disallows parameter properties in class constructors. <FEEDBACK>
+  'no-parameter-properties': false,
```

Comment:
I would leave this off for now. Might revisit in the future, but it can be a nice quick way to add properties, and is probably familiar to folks from CoffeeScript.